### PR TITLE
Add inset border style to input and output elements

### PR DESCRIPTION
**Pull request by catepalmer**

### CODE REVIEW

File: src/Editor.css [hunk 1]
```diff
@@ -3,8 +3,10 @@
 }
 
 .editor__textarea {
-  display: inline;
+  display: inline-block;
```
Comment (holloway) [inline src/Editor.css:6]: I guess this is a subtle aesthetic thing but I'm not sure if we want `inline-block` do we? Is this better with very long examples?

File: src/Editor.tsx [hunk 2]
```diff
@@ -42,7 +37,7 @@ const Editor = ({
           handleValueChanged(newValue);
         }}
       >
-        {value}
+        {defaultValue}
```
Comment (holloway) [inline src/Editor.tsx]: Should we rename this prop to be `value` ?

**Review by holloway — APPROVED** [review]

File: src/Editor.css (hunk 1 above)
Comment (catepalmer) [reply src/Editor.css:6]: Yeah, the reason I changed it was for long examples - when we have `border-style: inset;` and `display: inline;`, longer inputs look like this:

Screen Shot 2020-01-10 at 9 12 01 AM

Whereas with `display: inline-block;`, they look like this:

Screen Shot 2020-01-10 at 9 11 48 AM

Comment (holloway) [reply src/Editor.css:6]: Makes sense :+1:

Comment (holloway) [reply src/Editor.css:6]: I agree that `inline-block` is better, but fyi the earlier `inline` example could be fixed by increasing the `line-height`.

**Pull request merged**